### DISCUSSION TITLE
inconsistent editing behaviour between sidebar and top bar for parameters

### DISCUSSION THREAD

**Discussion by danielrosehill** [danny-avila/LibreChat#4918] [Q&A]
### What happened?

I'm not sure if this counts as a bug or if this is intended use, in which case I apologize. But if it is the latter, I thought I would explain what I'm experiencing. When I click on the parameters for the variant that I'm using of the model, and if I want to change the custom instructions or any variable for that matter, if I click on save as preset, irrespective of whether this is a new preset or an existing preset I'm trying to edit, it will assume that it's a new preset and bring me straight to the new preset menu item.  edit I see that GitHub limits video attachments to only 10 megabytes so I am simply sending this as a YouTube video instead it's unlisted privacy 

https://youtu.be/9hQh6teJkFQ

### Steps to Reproduce

Use a model, set custom parameters, try to edit the parameters while you're in a chat with the model. Click on save as new preset button and you will see this happen. 

### What browsers are you seeing the problem on?

Chrome

### Screenshots

two-minute video is at the following link 

https://youtu.be/9hQh6teJkFQ

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Hi Daniel,

Thanks for the video. This is intended, as the side bar use is for editing the current state of the conversation parameters, whereas "presets" simply apply an existing configuration to be the current state.

I can see how this is not intuitive, any suggestions for making it more intuitive are welcome.

**Comment by danielrosehill** [comment]
Hi Danny, 

Thanks for getting back to me and appreciate the clarification! The implementation actually makes total sense. 

I think rather than tweak a good system, the answer is probably just user education and documentation. I'm currently exploring all the features, but I would really love to help out in that respect in any way that I can (documentation, recording screencast, whatever could be helpful).
